pyo3-build-config/src/import_lib.rs

```diff
@@ -3,25 +3,28 @@
 use std::env;
 use std::path::PathBuf;
 
-use python3_dll_a::generate_implib_for_target;
+use python3_dll_a::ImportLibraryGenerator;
 
 use crate::errors::{Context, Result};
 
 use super::{Architecture, OperatingSystem, Triple};
 
-/// Generates the `python3.dll` import library for Windows targets.
+/// Generates the `python3.dll` or `pythonXY.dll` import library for Windows targets.
 ///
 /// Places the generated import library into the build script output directory
 /// and returns the full library directory path.
 ///
 /// Does nothing if the target OS is not Windows.
-pub(super) fn generate_abi3_import_lib(target: &Triple) -> Result<Option<String>> {
+pub(super) fn generate_import_lib(
+    target: &Triple,
+    py_version: Option<(u8, u8)>,
```

Comment:
A nitpick: Maybe using `Option<PythonVersion>` for `py_version` would simplify code at the call sites.